.github/workflows/_build-test-node.yml

```diff
@@ -30,32 +29,29 @@ jobs:
 
       - name: Install Rust toolchain
         uses: Cardinal-Cryptography/github-actions/install-rust-toolchain@v6
-        with:
```

Comment:
Is there any particular reason behind removing target?

Reply:
Yes - this wasm target was only required for `aleph-runtime` crate.